From fbdeccaff42291c07f894294d17f0ea93ba59975 Mon Sep 17 00:00:00 2001
From: Omar El-Domeiri <github@doesnotexist.com>
Date: Sun, 30 Jun 2024 03:34:58 +0000
Subject: [PATCH] Kconfig.nxp included only once in arch

---
 board/solidrun/imx8mp_solidrun/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/board/solidrun/imx8mp_solidrun/Kconfig b/board/solidrun/imx8mp_solidrun/Kconfig
index d7a81984fbe..7b025cf7ff4 100644
--- a/board/solidrun/imx8mp_solidrun/Kconfig
+++ b/board/solidrun/imx8mp_solidrun/Kconfig
@@ -12,6 +12,4 @@ config SYS_CONFIG_NAME
 config IMX_CONFIG
 	default "board/solidrun/imx8mp_solidrun/imximage-8mp-sd.cfg"
 
-source "board/freescale/common/Kconfig"
-
 endif
